forge-gui/res/cardsfolder/upcoming/resourceful_defense.txt

```diff
@@ -0,0 +1,13 @@
+Name:Resourceful Defense
+ManaCost:2 W
+Types:Enchantment
+
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Permanent.YouCtrl+HasCounters | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever a permanent you control leaves the battlefield, if it had counters on it, put those counters on target permanent you control.
+

```

Comment:
Empty lines would need to be removed from the final script.

Reply:
Done.